CQA Installing Proxy guide (#4932)
theforeman/foreman-documentation: guides/common/modules/con_performing-additional-configuration-on-smart-proxy-server.adoc

= Performing additional configuration on {SmartProxyServer}

[role="_abstract"]

> If you plan to register hosts to {SmartProxyServers}, configure them to provide remote execution, lifecycle environments, and power management functionalities.

Reviewer: "them" is ambiguous.
```suggestion
If you plan to register hosts to {SmartProxyServers}, configure your {SmartProxyServers} to provide remote execution, lifecycle environments, and power management.
```

> Additional {SmartProxyServer} configuration options include host registration and provisioning, remote execution, lifecycle environments, and power management, enabling extended functionality beyond basic installation.

Reviewer 2: Please rewrite with a user-focused perspective.

> You can perform additional {SmartProxyServer} configurations including host registration and provisioning, remote execution, lifecycle environments, and power management, enabling extended functionality beyond basic installation.

Reviewer 3: Starting with "You can perform additional {SmartProxyServer} configurations" does not really tell the user _why_ to follow this. I suggest to rephrase it:

```suggestion
If you plan to register hosts to {SmartProxyServers}, configure them to provide remote execution, lifecycle environments, and power management functionalities.
```
